docs/QbeastFormat.md

```diff
@@ -63,15 +63,15 @@ Here you can see the changes on the `AddFile` **`tags`** information
 }
 ```
 
-| Term           | Description                                                                                                                                  |
-|----------------|----------------------------------------------------------------------------------------------------------------------------------------------|
-| `revision`     | The metadata of the tree. Locates the particular tree in the space, which includes the min/max values for each column indexed and timestamp. |
-| `blocks`       | A list of the blocks that are stored inside the file. Can belong to different cubes.                                                         |
-| `cube`         | The serialized representation of the Cube.                                                                                                   |
-| `minWeight`    | The minimum weight of the block.                                                                                                             |
-| `maxWeight`    | The maximum weight of the block.                                                                                                             |
-| `elementCount` | The number of elements in the block.                                                                                                         |
-| `replicated`   | A flag that indicates if the block is replicated. Replication is **not available** from v0.6.0.                                                  |
+| Term           | Description                                                                                                    |
+|----------------|----------------------------------------------------------------------------------------------------------------|
+| `revision`     | **Locates a particular `min`-`max` range of the columns** in the space, and includes a Timestamp of the creation time. |
```

Comment:
This is interesting, but I'm not sure I understand. How does it determine what min-max range is relevant and what if there are multiple files touching that min-max range? Is this adding a timestamp to tags in the addFile entry or somewhere else?

Reply:
The min-max is determined by the data itself. 
Every append has a range. An append A1 may or may not trigger a Revision Change (new Revision R2) if the ranges of the batch are larger than the previous revision (R1). 
The timestamp is added to the Revision schema. Each file entry in the log contains only the ID for the Revision. (1 or 2 in the case above)

Reply:
Even the timestamp reference can be removed, it can be confusing to mention it when it's not the main characteristic of the revision, and the full schema is explained after.

What do you think of:

> A revision establishes the min-max range for each indexed column, amongst other metadata from a particular state of the index (creation timestamp, cube size...)